Edit `GITHUB_ORG_MANAGEMENT_POLICY.md` (#908)
nodejs/admin: GITHUB_ORG_MANAGEMENT_POLICY.md

# Node.js GitHub Organization Management Policy

The following GitHub Organizations are development resources under the direction
of the Node.js Technical Steering Committee (TSC):

* [`nodejs`](https://github.com/nodejs)
* [`pkgs`](https://github.com/pkgjs)

## Node.js Admin Repository

The [Node.js admin repository][nodejs/admin] serves as the
central location for managing our GitHub Organization administrative
activities. This repository is maintained by the TSC.

## Organization Roles

### Owners

Whether to grant Owner role is determined by optimizing
for the following conflicting requirements:

* Limiting access to reduce risk.
* Enabling individuals to move community work forward without undue delay.

When possible, automation and tools should be used to reduce the breadth of
access that needs to be provided in order to enable individuals to move
community work forward. As these tools are created, the groups to which

> Owner role are granted will be reduced.

Author: ```suggestion
Organization Owner role are granted will be reduced.
```

The following groups have the role "Owner" in our GitHub Organizations:

* **TSC voting members**.
* **OpenJS Director of Program Management**. The OpenJS Director of Program
  Management will limit their use of the access granted to accept GitHub terms
  and conditions. They will be added to the team called

>   [openjs-administrative-support](https://github.com/orgs/nodejs/teams/openjs-administrative-support)

Reviewer: Ah, this team doesn't exist, assuming it did at some point and I just recreated one yesterday as Ben was in 0 teams I wonder what happened:

```suggestion
  [openjs](https://github.com/orgs/nodejs/teams/openjs)
```

members to the organization when requested by a Working Group or team.

## Repositories

Any repository created under the our GitHub Organizations are considered to be
a project under the ownership of the OpenJS Foundation, and thereby subject
to the Intellectual Property and Governance policies of the Foundation.

Any member of one of our GitHub Organizations may request the management
of repositories within our GitHub Organizations by opening an issue in the
[Node.js admin repository][nodejs/admin]. The actions requested could be:

- Creating a new repository.
- Deleting an existing repository.
- Archiving an existing repository.
- Transferring a repository into or out of one of our GitHub Organizations.

Provided there are no objections from any TSC members raised in
the issue, such requests are approved automatically after 72 hours. If any
objection is made, the request shall not be completed without a vote from
the TSC to dismiss the objection.

In certain cases, OpenJS Cross Project Council and/or OpenJS Foundation Board
of Directors approval may also be required.

### Repository Team Ownership

When making a request to create a new repository, specify the team(s) that will

> have Maintain permission. If there is not an appropriate team to maintain a

Author: ```suggestion
have Maintain role. If there is not an appropriate team to maintain a
```

membership or block individuals. This is due largely to
limitations in the way GitHub permissions are structured.

> Remove inactive members from our GitHub organizations can happen without any formality.

Reviewer: ```suggestion
Removal of inactive members from our GitHub organizations can happen without any formality.
```
